doc/cookbook/source/examples/classifier/linear_svm.rst

```diff
@@ -0,0 +1,60 @@
+=============================
+Linear Support Vector Machine
+=============================
+
+Linear Support Vector Machine is a binary classifier which finds a hyper-plane such that the margins between the two classes are maximized. These margins can be described by the following equations:
+
+.. math::
+    {\bf w}^\top{\bf x} + b = 1
+
+    {\bf w}^\top{\bf x} + b = -1
+
+where :math:`{\bf w}` is vector of weights, :math:`{\bf x}` is feature vector and :math:`b` is bias.
+
+The distance between these two margins is :math:`\frac{2}{\|\mathbf{w}\|}`, so we want to minimize :math:`\|\mathbf{w}\|`. By using Lagrange Multipliers, we can show the solution to be:
+
+.. math::
+
+    \mathbf{w} = \sum_i \alpha_i y_i \mathbf{x}_i
+
+here most :math:`\alpha_i` turn out to be zero, which means that the solution is a sparse linear combination of the training features. The training features for which :math:`\alpha_i` are not zero are called support vectors.
+
+See Chapter 6 in :cite:`cristianini2000introduction` for a detailed introduction.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CBinaryLabels` as
+
+.. sgexample:: linear_svm.sg:create_features
+
+In order to run :sgclass:`CLibLinear`, we need to initialize some parameters like regularization coefficient and epsilon.
+
+.. sgexample:: linear_svm.sg:set_parameters
+
+We create an instance of the :sgclass:`CLibLinear` classifier by passing it regularization coefficient, features and labels. We also need to set a solver and epsilon.
+
+.. sgexample:: linear_svm.sg:create_instance
+
+Then we train and apply it to test data, which here gives :sgclass:`CBinaryLabels`.
+
+.. sgexample:: linear_svm.sg:train_and_apply
+
+We can extract :math:`{\bf w}` and :math:`b`.
```

Comment:
@karlnapf There is no function to extract the alphas. `CLibLinear` is derived from `CLinearMachine` and `get_alpha()` is available in `CKernelMachine`. In the ipython notebook for SVMs `w` and `b` are extracted to draw the decision boundary and `b` is non zero.